jsftemplating/src/main/java/com/sun/jsftemplating/el/PageSessionResolver.java

```diff
@@ -97,7 +97,20 @@ public Class<?> getType(ELContext elContext, Object base, Object property) {
 
     @Override
     public void setValue(ELContext elContext, Object base, Object property, Object value) {
-        checkPropertyFound(base, property);
+        if (base != null) {
```

Comment:
```
base The base object whose property value is to be set, or <code>null</code> to set a top-level variable.
```
Now you cannot set any value if the base is not null. Are we sure we don't have such cases?
Maybe we could add a log here at least ...

Reply:
I don't know. @avpinchuk , why do you think we shouldn't set values for base != null?

Reply:
@dmatej, we do not call `ELContext.setPropertyResolved()` and thus do not break resolver's chain. If we eliminate this check, we add a lot of a "garbage" properties to the PageSession scope (actually, View scope). IMHO

Reply:
@OndroMih, in opposite case you potentially add all properties to the PS (view) scope